_scripts/github_tag_push.sh

```diff
@@ -13,3 +13,8 @@ eval "$(ssh-agent -s)"
 # Add the key to the authentication agent
 DISPLAY=":0.0" SSH_ASKPASS="/tmp/askpass" /usr/bin/setsid ssh-add /tmp/git_deploy_key </dev/null
 
+## Push Github tags here 
+
```

Comment:
Is there supposed to be code here?

Reply:
Yes, the command that pushes tags into github should go there, so... `semantic_release` if I'm reading the documentation right.   Which I guess also publishes the package as well. 

So maybe the script should be renamed further...

Reply:
Ahhhhhhh I get it now. I didnt' realize that the kill had to happen after `semantic_release` happened.

In that case, it may make sense to go back to what we had _originally_ and the individual commands within `.travis.yml`, but this time under `deploy` like we have it (instead of `after_install` like before). Otherwise it means we have to make the `semantic-release` call in here and name it `semantic_release.sh` or something. But I'd _really_ like to have the `semantic-release` call visible in `.travis.yml`.

What do you think?

Reply:
We can try something like 
```
deploy: 
  script:
    - bash _scripts/deploy_key_decrypt.sh
    - semantic-release 
    - pkill ssh-agent
```

But I am not entirely sure that it will progress past the decryption step without killing the agent - I'm fine with trying it to see though.

Reply:
not ideal, but it might work. We could run `npm-run-all --sequential <start-ssh-agent><ci-yaml-script> <kill-ssh-agent>`

Reply:
`npm-run-all` isn't a random CI runner. It just runs NPM tasks. So we'll have to create tasks for each of those, which would be annoying

(I've wanted to do this a couple of times now 😉)

Reply:
@dtardoin-eb yeah I'm fairly certain that it won't get pass the decryption step either. That basically why #20 didn't work. I think it's gotta be all-in or all-out, and my vote is all-out 😄